### PR TITLE
[RUN-9296] added to RN only with link to Redoc

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

**Comment by jasonnovichRunAI** [comment]
> 

According to Noa F. we have not exposed the UI for this and it is only available through the API - as per Tal Levy. We do not have any of the "swagger" APIs in our regular doc site (only a link to the APIs). There really is no where else to mention it that I can think of. If you have a place let me know.

**Comment by gal-revach** [comment]
@jasonnovichRunAI ok, that makes sense, so let's keep it in the API docs and in what's new
once we'll expose this configuration in the UI we can also find a place to mention this on the regular docs site

**Comment by jasonnovichRunAI** [comment]
> @jasonnovichRunAI ok, that makes sense, so let's keep it in the API docs and in what's new once we'll expose this configuration in the UI we can also find a place to mention this on the regular docs site

If this ok please click on the approve button. It is in the "files changed", "review changes" (dropdown) and click approve.

**Comment by gal-revach** [comment]
@jasonnovichRunAI added a comment on the commit itself as well, let me know what you think

File: docs/home/whats-new-2-13.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Added to the Overview and the Analytics dashboards, a dropdown filter for node p
 Added `suspend`/`stop` to [Submitting Workloads via HTTP/REST](../developer/cluster-api/submit-rest.md).
 
 <!-- RUN-7757/9296 Custom logo in UI -->
+Added support for custom logos in SaaS, and self-hosted environments using the Admin API. For more information, see [Upload tenant logo](https://app.run.ai/api/docs#tag/Tenant/operation/upload_tenant_logo){target=_blank}.
```
[the remark this answers is not in the document]
Comment (gal-revach) [reply docs/home/whats-new-2-13.md:71]: Looks good, thanks

**Comment by jasonnovichRunAI** [comment]
Can you please approve the changes?
Thanks!

File: docs/home/whats-new-2-13.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ Added to the Overview and the Analytics dashboards, a dropdown filter for node p
 Added `suspend`/`stop` to [Submitting Workloads via HTTP/REST](../developer/cluster-api/submit-rest.md).
 
 <!-- RUN-7757/9296 Custom logo in UI -->
+Added support for uploading a custom company logo to be displayed in the UI. for both SaaS and self-hosted environments. Use the  [Upload tenant logo](https://app.run.ai/api/docs#tag/Tenant/operation/upload_tenant_logo){target=_blank} endpoint in the Admin API.
```
Comment (jasonnovichRunAI) [inline docs/home/whats-new-2-13.md:71]:
```suggestion
Added support for uploading a custom company logo to be displayed in the UI for both SaaS and self-hosted environments. Use the  [Upload tenant logo](https://app.run.ai/api/docs#tag/Tenant/operation/upload_tenant_logo){target=_blank} endpoint in the Admin API.
```

**Review by gal-revach — APPROVED** [review]

**Pull request merged**